dvc/repo/fetch.py

```diff
@@ -85,44 +102,51 @@ def fetch(
     return downloaded
 
 
-def _fetch_external(self, repo_url, repo_rev, files, jobs):
-    from dvc.external_repo import external_repo
-    from dvc.objects import save
-    from dvc.objects.stage import stage
-    from dvc.path_info import PathInfo
-    from dvc.scm.base import CloneError
+def _fetch_naive_objs(repo, objs, **kwargs):
```

Comment:
Once the remote push/pull are unified into ODB save, we will be able to get rid of these separate fetch helpers

Reply:
Just  to clarify, you mean that all three will go away, meaning that default odb, custom odb(e.g. for externals, or future per-output remotes) and git odb will all have the same fetching logic, right?

Reply:
Yes that's correct